.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,5 @@
+---
+name: Question
+about: Question for TOC or Steering Committee
+title: ""
+---
```

Comment:
That sounds like a good but orthogonal project.

It might be worth seeing if we can get @macruzbar to summarize some triage best-practices gleaned from across the WGs, so we don't need to design our own.